docs/reference/tasking-reference.md

```diff
@@ -74,43 +70,36 @@ The `get_feasibility()` function returns a list of feasibility studies for taski
-
-
-<h5> Returns </h5>
+<h5> Arguments </h5>
 
-| Type   | Description                    |
-| ------ | ------------------------------ |
-| `list` | A list of feasibility studies. |
+| Argument         | Overview                                                                                                                                                                                                                                                                                     |
+| ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `feasibility_id` | **str**<br/>The feasibility study ID.                                                                                                                                                                                                                                                        |
+| `workspace_id`   | **str**<br/>The workspace ID. Use to get objects from a specific workspace. Otherwise, objects from the entire account will be returned.                                                                                                                                                     |
+| `order_id`       | **str**<br/>The order ID.                                                                                                                                                                                                                                                                    |
+| `decision`       | **List[str]**<br/>The status of feasibility studies. The allowed values:<br/><ul><li>`NOT_DECIDED`</li><li>`ACCEPTED`</li></ul>                                                                                                                                                              |
+| `sortby`         | **str**<br/>Arrange elements in the order specified in `descending` based on a chosen field. The default value is `createdAt`.                                                                                                                                                               |
+| `descending`     | **bool**<br/>Determines the arrangement of elements:<br/><ul><li>`True`: arrange elements in descending order based on the field specified in `sortby`.</li><li>`False`: arrange elements in ascending order based on the field specified in `sortby`.</li></ul>The default value is `True`. |
 
 <h5> Example </h5>
 
 ```python
 tasking.get_feasibility(
-    feasibility_id=None,
     workspace_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
-    order_id=None,
     decision="NOT_DECIDED",
-    sortby="createdAt",
-    descending=False
+    sortby="updatedAt",
```

Comment:
nice!

Reply:
did you verify that `updatedAt` field exists?

Reply:
I did